modules/ROOT/pages/database-administration/standard-databases/seed-from-uri.adoc

```diff
@@ -0,0 +1,229 @@
+:page-role: enterprise-edition
+:description: How to create a database using a seed from URI. 
```

Comment:
@renetapopova, will we need a `page-aliases` attr here to bind this page with 4.4?

Reply:
Yes, I think so.